app/models/feed.rb

```diff
@@ -81,17 +82,14 @@ def feed_episodes
     include_in_feed = []
     feed_max = display_episodes_count.to_i
     filtered_episodes.each do |ep|
-      next unless include_episode_categories?(ep)
```

Comment:
+1 ... makes sense to move these checks into `filtered_episodes`.

And I was surprised to see that `Episode.include_in_feed?` is making some completely unrelated checks ... if the media is ready.  I'm guessing the name of that method predates the `Feed` model.